packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/FLTQuickActionsPluginTests.m

```diff
@@ -0,0 +1,151 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Flutter;
+@import quick_actions_ios;
+@import quick_actions_ios.Test;
+@import XCTest;
+#import <OCMock/OCMock.h>
+
+@interface FLTQuickActionsPluginTests : XCTestCase
+
+@end
+
+@implementation FLTQuickActionsPluginTests
+
+// A dummy `UIApplicationShortcutItem`.
+- (UIApplicationShortcutItem *)searchTheThingShortcutItem {
+  return [[UIApplicationShortcutItem alloc]
+           initWithType:@"SearchTheThing"
+         localizedTitle:@"Search the thing"
+      localizedSubtitle:nil
+                   icon:[UIApplicationShortcutIcon
+                            iconWithTemplateImageName:@"search_the_thing.png"]
+               userInfo:nil];
+}
+
+// A dummy raw shortcut item.
+- (NSDictionary<NSString *, NSString *> *)searchTheThingRawItem {
+  return @{
+    @"type" : @"SearchTheThing",
+    @"localizedTitle" : @"Search the thing",
+    @"icon" : @"search_the_thing.png",
+  };
+}
+
+- (void)testHandleMethodCall_setShortcutItems {
+  FlutterMethodCall *call =
+      [FlutterMethodCall methodCallWithMethodName:@"setShortcutItems"
+                                        arguments:@[ self.searchTheThingRawItem ]];
+
+  FLTQuickActionsPlugin *plugin =
+      [[FLTQuickActionsPlugin alloc] initWithChannel:OCMClassMock([FlutterMethodChannel class])];
+  XCTestExpectation *resultExpectation =
+      [self expectationWithDescription:@"result block must be called."];
+  [plugin handleMethodCall:call
+                    result:^(id _Nullable result) {
+                      XCTAssertNil(result, @"result block must be called with nil.");
+                      [resultExpectation fulfill];
+                    }];
+  [self waitForExpectationsWithTimeout:1 handler:nil];
```

Comment:
nits:
Maybe let's increase the timeout a little bit more :p

Reply:
I think unit tests should be fine.